code/modules/mob/living/login.dm

```diff
@@ -14,7 +14,11 @@
 	if(ticker && ticker.mode)
 		switch(ticker.mode.name)
 			if("sandbox")
-		/*		CanBuild()
+				CanBuild()
+
+		if (iscultist(src) || iswizard(src) || isnukeop(src))
```

Comment:
Have an IsInFaction check or something, so it's more modular than sticking a macro to see if the person is in 'one of these factions'

And don't have the IsInFaction check just look at every faction macro